src/main/java/com/yo1000/sonarqube/collaboration/FizzBuzz.java

```diff
@@ -13,6 +13,12 @@ public String buzz(int value) {
     }
 
     public String fizzBuzz(int value) {
+        if (true) {
+            if (false) {
+                int a = 1;
+            }
+            // TODO: Do nothing
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://dev.yo1000.com/coding_rules#rule_key=squid%3AS1135)